components/generic/Button/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import { MouseEventHandler } from 'react';
+import { getButtonSize, ButtonSize } from './getButtonSize';
+import { getButtonColor, ButtonColor } from './getButtonColor';
+import { clsx as cx } from 'clsx';
+
+type ButtonProps = {
+  name: string;
+  color: ButtonColor;
+  size: ButtonSize;
+  isLight?: boolean;
```

Comment:
You can use the same convention as in heading `fontWeight`, I think it would be wise to describe what's `className` used for in README.md.

```suggestion
  isLight?: boolean;
```